src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -91,7 +91,7 @@ public without sharing class UTIL_CustomSettingsFacade {
      * Private methods that will configure the default org-level custom settings.
      **/
     private static void configSettings(Hierarchy_Settings__c hs) {
-		hs.Account_Processor__c = 'Administrative';
+		hs.Account_Processor__c = UTIL_Describe.getRecTypesMapByDevName('Account').get('Administrative');
```

Comment:
So, what happens if the recordtype name is something other that Administrative? Administrativo?